terracumber_config/tf_files/MLM-5.1-SLE-update-NUE.tf

```diff
@@ -0,0 +1,232 @@
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = string
+}
+
+variable "SCC_PASSWORD" {
+  type = string
+}
+
+variable "SERVER_CONTAINER_REPOSITORY" {
+  type = string
+}
+
+variable "PROXY_CONTAINER_REPOSITORY" {
+  type = string
+}
+
+variable "SERVER_CONTAINER_IMAGE" {
+  type = string
+  default = ""
+}
+
+variable "GIT_USER" {
+  type = string
+  default = null
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null
+}
+
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.8.1"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://suma-11.mgr.suse.de/system"
+}
+
+module "base" {
+  source            = "./modules/base"
+
+  cc_username       = var.SCC_USER
+  cc_password       = var.SCC_PASSWORD
+  product_version   = "5.1-released"
+  name_prefix       = "mlm-su-51-"
+  use_avahi         = false
+  domain            = "mgr.suse.de"
+  images            = [ "sles15sp6o", "opensuse155o", "opensuse156o", "slemicro55o" ]
+
+  mirror            = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite         = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br1"
+  }
+}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base.configuration
+  name               = "server"
+  image              = "slemicro55o"
+  provider_settings = {
+    mac                = "aa:b2:93:01:02:79"
+    data_pool          = "ssd"
+  }
+
+  main_disk_size        = 100
+  repository_disk_size  = 1000
+  database_disk_size    = 150
+  runtime               = "podman"
+  // Temporary workaround to see if we pass proxy stage. Also needs to be updated on next MU
+  container_repository  = var.SERVER_CONTAINER_REPOSITORY
+  container_image       = var.SERVER_CONTAINER_IMAGE
+  container_tag         = "latest"
+  server_mounted_mirror = "minima-mirror-ci-bv.mgr.suse.de"
+
+  auto_accept                    = false
+  monitored                      = true
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
+  disable_download_tokens        = false
+  ssh_key_path                   = "./salt/controller/id_ed25519.pub"
+  from_email                     = "root@suse.de"
+
+  //server_additional_repos
+
+}
+
+module "proxy_containerized" {
+  source             = "./modules/proxy_containerized"
+  base_configuration = module.base.configuration
+  name               = "proxy"
+  image              = "slemicro55o"
```

Comment:
For MLM 5.1, we need either SL Micro 6.1 or SLES 15 SP7 as host OS.

Reply:
fixed